static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -175,15 +160,15 @@ const PanelItemCenter = styled(PanelItem)`
   cursor: pointer;
 `;
 
-const Container = styled('div')<ContainerProps>`
-  display: flex;
+const Container = styled('button')<ContainerProps>`
+  display: inline-flex;
+  width: 100%;
+  border: none;
+  background: ${p => p.theme.white};
```

Comment:
```suggestion
  background: transparent;
```

`transparent` background might be a better value to reset the button styles.

I'm thinking that since there was no explicit background picked before we don't need to pick a color when we reset things.